Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -581,7 +595,17 @@ public final class PackageBuilder {
                 successors += pluginUsages.compactMap({
                     switch $0 {
                     case .plugin(let name, let package):
-                        return (package == nil) ? potentialModuleMap[name] : nil
```

Comment:
nit: `case .plugin(let name, let Some(package)):` and case .plugin(let name, .none):